### PR TITLE
lint & format `io/{tests, util}/*.py` part 2

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR formats and lints `io/{tests, util}/*.py` –  part 2 of $n$

### CODE REVIEW

File: libpysal/io/util/tests/test_shapefile.py [hunk 1]
```diff
@@ -282,23 +288,29 @@ def test___init__(self):
...
+            b"\x14@\x00\x00\x00\x00\x00\x00\x14@"
         )
         expected = {"X": 5, "Y": 5, "Z": 5, "M": 5, "Shape Type": 11}
         assert expected == PointZ.unpack(dat)
 
 
-class TestPolyLineZ:
-    def test___init__(self):
-        pytest.raises(NotImplementedError, PolyLineZ)
+# class TestPolyLineZ:
```
Comment (jGaboardi) [inline libpysal/io/util/tests/test_shapefile.py:308]: This test was being overwritten by the following one and never actually tested. When actually run it does not raise the `NotImplementedError`. Commenting out.

**Review by martinfleis — APPROVED** [review]
Are we getting anywhere near the end of this linting journey?

**Comment by jGaboardi** [comment]
> Are we getting anywhere near the end of this linting journey?

See https://github.com/pysal/libpysal/issues/589

**Comment by jGaboardi** [comment]
≈75% I'd say.

**Pull request merged**